### PR TITLE
ADAPT-3119: Checkbox component

### PR DESCRIPTION
**Pull request by Lyafel**
# Summary
- Created and themed the checkbox component. Pulled in ADAPT-3250 which is unmerged yet so depending on that one. 

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@Lyafel I'm out today but will review tomorrow. Thanks!!

File: src/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            focus-visible:!su-ring-offset-0 focus-visible:!su-ring-8 focus-visible:!su-border-digital-blue
+            hocus:!su-ring-offset-0 hocus:!su-ring-8
+            focus-visible:!su-ring-digital-blue-light focus-visible:!su-ring-opacity-50
+            focus-visible:su-bg-digital-blue-light focus-visible:su-bg-opacity-50
+            focus-visible:!su-border-digital-blue focus-visible:!su-border-opacity-50 focus-visible:!su-border-[3px]
+            focus:!su-ring-digital-blue-light focus:!su-ring-opacity-50 focus:su-bg-digital-blue-light
+            focus:su-bg-opacity-50  focus:!su-border-digital-blue focus:!su-border-opacity-50 focus:!su-border-[3px]
+            hover:!su-ring-digital-blue-light hover:!su-ring-opacity-20 hover:su-bg-digital-blue-light
+            hover:!su-border-digital-blue hover:!su-border-opacity-70 hover:su-bg-opacity-20
+            checked:!su-ring-2 checked:!su-border-digital-blue
```
Comment (sherakama) [inline src/Checkbox/Checkbox.js:77]: That is a lot of `!` marks. Is there something in our base styles that is making this difficult? Also, I am not seeing our `su-checkbox` TW component here. Was it not useful?

https://github.com/SU-SWS/decanter/blob/v7/src/plugins/components/form/form-elements.js#L7

**Comment by sherakama** [comment]
@Lyafel please fix the failing jest test.

File: src/Checkbox/Checkbox.js (hunk 1 above)
Comment (Lyafel) [reply src/Checkbox/Checkbox.js]: @sherakama Removed the !, those weren't meant to get into the pr, thanks for catching that.  
As for the su-checkbox, there's not much of use in that class.

**Comment by sherakama** [comment]
@Lyafel This is now on hold. Please wait until further notice to where this work is going. We are going to be retiring this project and moving the work directly into projects.

**Pull request closed without merging**